t/lib/t/MusicBrainz/Server/Controller/Area/Aliases.pm

```diff
@@ -33,7 +33,7 @@ test 'Area alias appears on alias page content and on JSON-LD' => sub {
 
     page_test_jsonld $mech => {
         '@id' => 'http://musicbrainz.org/area/106e0bec-b638-3b37-b731-f53d507dc00e',
-        'alternateName' => ["\x{30aa}\x{30fc}\x{30b9}\x{30c8}\x{30e9}\x{30ea}\x{30a2}"],
+        'alternateName' => ["\N{KATAKANA LETTER O}\N{KATAKANA-HIRAGANA PROLONGED SOUND MARK}\N{KATAKANA LETTER SU}\N{KATAKANA LETTER TO}\N{KATAKANA LETTER RA}\N{KATAKANA LETTER RI}\N{KATAKANA LETTER A}"],
```

Comment:
In this commit, I'm not reviewing any of the changes under t/, because I assume the tests would fail if the wrong characters were used.